Extension/src/main.ts

```diff
@@ -368,37 +173,38 @@ function handleError(installBlob: InstallBlob, error: any): void {
         }
 
         if (packageError.errorCode) {
-            installBlob.telemetryProperties['error.errorCode'] = removePotentialPII(packageError.errorCode);
+            installBlob.telemetryProperties['error.errorCode'] = util.removePotentialPII(packageError.errorCode);
         }
     } else {
         errorMessage = error.toString();
-        installBlob.telemetryProperties['error.toString'] = removePotentialPII(errorMessage);
+        installBlob.telemetryProperties['error.toString'] = util.removePotentialPII(errorMessage);
     }
 
     // Show the actual message and not the sanitized one
-    if (installBlob.stage == "downloadPackages") {
+    if (installBlob.stage == InstallBlobStage.downloadPackages) {
         channel.appendLine("");
     }
-    channel.appendLine(`Failed at stage: ${installBlob.stage}`);
+    channel.appendLine(`Failed at stage: ${InstallBlobStage[installBlob.stage]}`);
     channel.appendLine(errorMessage);
     channel.appendLine("");
     channel.appendLine(`If you work in an offline environment or repeatedly see this error, try downloading a version of the extension with all the dependencies pre-included from https://github.com/Microsoft/vscode-cpptools/releases, then use the "Install from VSIX" command in VS Code to install it.`);
```

Comment:
It would be nice if it would omit this message if the error occurred during offline installation...I was going to fix that, but I didn't really think it was worth it since we don't expect the offline install to fail.